### PR TITLE
Update contents mobile version

### PR DESCRIPTION
**Pull request by Bearbbcjtc**
Update contents  and revise the mobile version

### CODE REVIEW

**Comment by alan-walsh** [comment]
@Bearbbcjtc I do have one question: I noticed that with this new version, when I click on the ARDaC logo in the top left corner, the menu goes away. I confirmed that this is NOT the case with the current version, where the menu remains. Is this intentional?

**Comment by Bearbbcjtc** [comment]
> @Bearbbcjtc I do have one question: I noticed that with this new version, when I click on the ARDaC logo in the top left corner, the menu goes away. I confirmed that this is NOT the case with the current version, where the menu remains. Is this intentional?

@alan-walsh Oh I never found this issue before. Let me look into it and fix it then update the submission again. Thanks a lot for pointing that issue out!

**Comment by Bearbbcjtc** [comment]
@alan-walsh Hi, I have fixed the issue and updated the submission, could you please review it again? Thanks!

File: src/index.html [hunk 1]
```diff
@@ -314,7 +355,7 @@
 24.0pt;font-family:"Segoe UI",sans-serif;color:#24292F'>Publications</span></b></p>
     <ul type=disc>
         <li class=MsoNormal style='color:#24292F;background:white'><span
-                style='font-family:"Segoe UI",sans-serif'>N Jin, Z Li, C Kettler, B Yang, W Tu, J Su, �ARDaC Common Data Model Facilitates Data Dissemination and Enables Data Commons for Modern Clinical Studies.� Studies in Health Technology and Informatics. 310:3-7(<b>2024</b>). doi: 10.3233/SHTI230916. PMID: 38269754.
+                style='font-family:"Segoe UI",sans-serif'>N Jin, Z Li, C Kettler, B Yang, W Tu, J Su, ?ARDaC Common Data Model Facilitates Data Dissemination and Enables Data Commons for Modern Clinical Studies.? Studies in Health Technology and Informatics. 310:3-7(<b>2024</b>). doi: 10.3233/SHTI230916. PMID: 38269754.
```
Comment (alan-walsh) [inline src/index.html]:
```suggestion
                style='font-family:"Segoe UI",sans-serif'>N Jin, Z Li, C Kettler, B Yang, W Tu, J Su, "ARDaC Common Data Model Facilitates Data Dissemination and Enables Data Commons for Modern Clinical Studies." Studies in Health Technology and Informatics. 310:3-7(<b>2024</b>). doi: 10.3233/SHTI230916. PMID: 38269754.
```

**Comment by alan-walsh** [comment]
LGTM
🚢🚢🚢🚢🚢

**Review by alan-walsh — APPROVED** [review]

**Pull request merged**